packages/idos-sdk-js/src/lib/idos.ts

```diff
@@ -62,12 +62,14 @@ export class idOS {
   }
 
   static async init(params: InitParams): Promise<idOS> {
+    if (idOS.instance) return idOS.instance;
```

Comment:
what's the case where Dev need to has two active idOS instances (in the same container or otherwise)
without resetting the first and initializing other one?